### DISCUSSION TITLE
Playwright -m doing nothing

### DISCUSSION THREAD

**Discussion by beardviking** [davidteather/TikTok-Api#738] [Q&A] [answered]
I'll admit I'm fairly new, but when adding `python -m playwright install ` it doesn't do anything and returns a blank cd. I'm not exactly sure what I'm doing wrong. I followed his video step by step and on his video, chromium and whatever else is installing. Any idea what's wrong?

**Comment by zulc22** [comment, marked as answer]
if it doesn't do anything and doesn't return any errors, playwright already has what it needs, and the install command does not need to be run.
